File: 1980-find-unique-binary-string/1980-find-unique-binary-string.py
class Solution:
  def findDifferentBinaryString(self, nums: List[str]) -> str:
    n, s = len(nums[0]), set(nums)
    def backtrack(v = ''):
      if len(v) == n:
        return '' if v in s else v
      return backtrack(v + '1') or backtrack(v + '0')
    return backtrack()
        

File: 1980-find-unique-binary-string/1980-find-unique-binary-string.go
func findDifferentBinaryString(nums []string) string {
  bytes := []byte{}
  for i := range(len(nums)) {
    if nums[i][i] == 49 {
      bytes = append(bytes, 48)
    } else {
      bytes = append(bytes, 49)
    }
  }
  return string(bytes)
}